lib/config.js

```diff
@@ -59,49 +59,78 @@ function isFilePath(filePath) {
 }
 
 /**
- * Load and parse a JSON config object from a file.
+ * Check if item is an javascript object
+ * @param {*} item object to check for
+ * @returns {boolean} True if its an object
+ * @private
+ */
+function isObject(item) {
```

Comment:
I am not sure if we need that. As this function is sufficient enough for our usecase. It also saves an additional dependency.
Am I missing any usecase?